css/css-borders/tentative/border-radius-side-shorthands/border-radius-side-shorthands-001.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Borders and Box Decorations 4 Test: Physical 'border-*-radius' shorthands set the two radii of one side.</title>
+<link rel="author" title="Sebastian Zartner" href="mailto:sebastianzartner@gmail.com">
+<link rel="help" href="https://drafts.csswg.org/css-borders-4/#corner-sizing">
+<link rel="match" href="../../reference/border-radius-side-shorthands-ref.html">
```

Comment:
> having the references so far away from the tests

That is because he had to put them into a /tentative/ folder. And such folder is bound to be removed once the spec is more reliable, is less "drafty".